src/components/StudyInfo/CavaticaCounts.js

```diff
@@ -23,7 +30,9 @@ const CavaticaCounts = ({projects, title, hideIcon, wrap, isResearch}) => {
             : `/study/${title}/cavatica`
         }
         onClick={e => e.stopPropagation()}
-        className={hideIcon && projects.length === 0 ? 'text-red' : null}
+        className={
+          hideIcon && projects.length === 0 && showWarning ? 'text-red' : null
```

Comment:
I don't think this changes the path color of the svg. It looks like maybe the fill is hard coded onto it. You may be able to get around it by changing the `fill` on the `path` tag.